setup.py

```diff
@@ -5,11 +5,26 @@
 
 ext_modules = [
     Extension('ffm', ['fastFM/ffm.pyx'],
-              libraries=['m', 'fastfm'],
-              library_dirs=['fastFM/', 'fastFM-core/bin/'],
+              libraries=['m', 'fastfm', 'openblas'],
+              library_dirs=['fastFM/', 'fastFM-core/bin/',
+                            'fastFM-core/externals/OpenBLAS/',],
               include_dirs=['fastFM/', 'fastFM-core/include/',
                             'fastFM-core/externals/CXSparse/Include/',
-              numpy.get_include()])]
+                            'fastFM-core/externals/OpenBLAS/',
+                            numpy.get_include()]),
+    Extension('ffm2', ['fastFM/ffm2.pyx'],
+              libraries=['fastFMd', 'pthread', 'glog'],
```

Comment:
linking against `glog` and `pthread` shouldn't be necessary as soon as fastFMd is properly statical linked? I'm worried about binary compatibility issues if `fastFMd` and `pthread, glog` aren't build on the same machine.